src/poly/multiopen.rs

```diff
@@ -279,21 +321,9 @@ fn test_roundtrip() {
             &params,
             &mut transcript,
             std::iter::empty()
-                .chain(Some(VerifierQuery {
-                    point: x,
-                    commitment: &a,
-                    eval: avx,
-                }))
-                .chain(Some(VerifierQuery {
-                    point: x,
-                    commitment: &b,
-                    eval: avx, // NB: wrong!
-                }))
-                .chain(Some(VerifierQuery {
-                    point: y,
-                    commitment: &c,
-                    eval: cvy,
-                })),
+                .chain(Some(VerifierQuery::new_commitment(&a, x, avx)))
+                .chain(Some(VerifierQuery::new_commitment(&b, x, avx))) // NB: wrong!
```

Comment:
Fix this, or remove the comment.

Reply:
This is supposed to be wrong; it's a test checking that the bug is detected.